### PR TITLE
onboarding first page implementation

### PR DESCRIPTION
**Pull request by kiroj305**
- Added the very first screen
- Customized PylonsButton which will be used in all screens (note: doesn't have any click effect, will look at it later)

### CODE REVIEW

File: lib/widgets/pylons_button.dart [hunk 1]
```diff
@@ -1,19 +1,29 @@
...
+      {Key? key,
+      required this.btnText,
+      required this.showArrow,
+      required this.onPressed})
+      : super(key: key);
 
   @override
   Widget build(BuildContext context) {
-    return TextButton.icon(
+    return /*TextButton.icon(
```
Comment (mustafapylons) [inline lib/widgets/pylons_button.dart]: What's the purpose of leaving all of this commented? Is this a to-do later kind of item?

**Review by mustafapylons — COMMENTED** [review]
Outside of a commented block of code which requires some clarification looks good to me.

File: lib/widgets/pylons_button.dart (hunk 1 above)
Comment (kiroj305) [reply lib/widgets/pylons_button.dart]: Forgot to remove them. thanks for pointing this out.

**Review by mustafapylons — APPROVED** [review]
Looks good to me now after removing commented code.

File: lib/widgets/pylons_button.dart [hunk 2]
```diff
@@ -23,6 +33,43 @@ class PylonsButton extends StatelessWidget {
...
+        GestureDetector(
+      child: Material(
+          color: Colors.transparent,
+          elevation: 10,
+          child: SizedBox(
+            width: 0.5.sw,
+            height: 0.12.sw,
+            child: Stack(
+              children: [
+                SvgPicture.asset("assets/images/svg/rectangular_button.svg",
```
Comment (kjawadDeveloper2) [inline lib/widgets/pylons_button.dart]: Add the svg in a constant file.

File: lib/widgets/splash_widget.dart [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+class SplashWidget extends StatelessWidget {
+  const SplashWidget({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return SizedBox(
+        height: 1.sh,
+        width: 1.sw,
+        child: SvgPicture.asset("assets/images/svg/splash.svg",
```
Comment (kjawadDeveloper2) [inline lib/widgets/splash_widget.dart]: Add svg in a constant file.

**Review by kjawadDeveloper2 — CHANGES_REQUESTED** [review]
These small changes. But the code quality was a lot good .

**Comment by kiroj305** [comment]
@kjawadDeveloper2 , the issues you pointed out are now fixed.

**Review by kjawadDeveloper2 — APPROVED** [review]
Looks good from my side.

**Pull request merged**